src/elements/pages/dev/game/games.ts

```diff
@@ -259,30 +259,32 @@ export default class DevGames extends LitElement {
 
 	renderBody() {
 		return html`
-			${this.data.groups.length
-				? html`<div id="groups-list">
-						${repeat(
-							this.data.groups,
-							g => g.groupId,
-							g => this.renderGroup(g)
-						)}
-				  </div>`
-				: null}
-			${when(
-				global.currentIdentity.isRegistered && global.currentIdentity.isAdmin,
-				() =>
-					html` <div
-						class="dashed-border-button flex justify-center items-center hover:cursor-pointer w-full h-32 place-content-center text-[#d1d1d1]  hover:text-white hover:bg-button-bg-hover-color"
-						@click=${this.openGroupModal.bind(this)}
-						@mouseenter=${() => (this.createGroupHovered = true)}
-						@mouseleave=${() => (this.createGroupHovered = false)}
-					>
-						<div class="font-bold text-lg pb-0.5 pr-2">
-							<e-svg src="solid/plus"></e-svg>
-						</div>
-						<div class="font-bold text-lg text-center">New Developer Group</div>
-					</div>`
-			)}
+			<div class="max-w-contentwidth px-[10px] md:px-5 lg:px-0">
```

Comment:
I'd suggest sticking with predefined values form tailwind theme - instead of using `10px`, find something very close to 10px.